src/xpcc/architecture/platform/driver/core/cortex/platform.macros

```diff
@@ -0,0 +1,47 @@
+%# coding: utf-8
+%# Copyright (c) 2011-2016, Roboterclub Aachen e.V.
+%# All Rights Reserved.
+%#
+%# The file is part of the xpcc library and is released under the 3-clause BSD
+%# license. See the file `LICENSE` for the full license governing this code.
+%# ----------------------------------------------------------------------------
+
+%# This File includes platform specific macros for the generic Cortex-M startup
+
+%# WARNING: Since this is the first code executed after a reset, you do not
+%#          have access to _any_ data stored in RAM, since it has not yet been
+%#          initialized.
+%#          In the worst case you won't even have access to the stack, if the
+%#          memory containing the stack is not physically enabled yet.
+%#          In that case, consider using inline assembly to manage stack access
+%#          manually, until the memory is enabled.
```

Comment:
Can't we execute this code after initializing the main stack?
I'd love to keep our reliance on undefined behavior to a minimum, especially in this case, where the execution order isn't obvious, as this code is pasted into the startup function.